apps/openmw/mwrender/objectpaging.cpp

```diff
@@ -428,11 +428,9 @@ namespace MWRender
                                 continue;
 
                             if (std::find(cell->mMovedRefs.begin(), cell->mMovedRefs.end(), ref.mRefNum) != cell->mMovedRefs.end()) continue;
-                            Misc::StringUtils::lowerCaseInPlace(ref.mRefID);
```

Comment:
Shouldn't all the `Misc::StringUtils::lowerCaseInPlace` removal be done in a dedicated PR, all at once?

Reply:
We will have to review and adapt individually for the time being because some of our code still requires this practice.